src/makeRobot.js

```diff
@@ -38,7 +38,73 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, `
+      + `chip version: ${this.version}, `
+      + `wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(steps = 1) {
+      if (steps <= 0) {
+        this.coords.y += 0;
+      } else {
+        this.coords.y += steps;
+      }
+
+      return this;
+    },
```

Comment:
```suggestion
    goForward(steps = 1) {
      if (steps > 0) {
        this.coords.y += steps;
      }

      return this;
    },
```
`+= 0` никак не меняет же значение

Reply:
Исправил. Я понимаю, что не меняет, вот только уже не помню, почему написал именно так))) Наверное, хотел от чего-то перестраховаться (хотя от чего?) Нужно комментарии в коде таки оставлять)))
Возможно, так написал, чтобы легко можно было поменять поведение при отрицательном или нулевом значении параметра (если бы это теоретически понадобилось сделать). Кажется, так я и размышлял))) Перемудрил в общем.